website/docs/getting-started/configuration.md

```diff
@@ -145,7 +145,7 @@ analyzer:
 
 will work both for the analyzer and for this plugin.
 
-If you want a specific rule to ignore files, you can configure `exclude` entry for it. For example,
+If you want a specific rule or anti-pattern to ignore files, you can configure `exclude` entry for it. For example,
```

Comment:
Let's have a separate entry for anti-pattern excludes since it has a different config.